src/lev-report/UsageCounter.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const StyledCounter = styled.div`
```

Comment:
I'd call this `StyledDiv` or just `Container` or something. It's not a 'Counter' at this point.

Reply:
Changed

Reply:
Have you commited and pushed the change? It doesn't seem to have updated.

Reply:
No, I'm shortly going to after I fix another issue...